src/metax_api/api/rest/base/serializers/file_serializer.py

```diff
@@ -160,6 +161,19 @@ def validate_file_path(self, value):
 
         return value
 
+    def validate_checksum_algorithm(self, value):
+        """
+        Ensure that used checksum_algorithm is one of the configured in settings file.
+        """
+
+        checksum_algorithm = value.lower().replace('-', '')
```

Comment:
No need to do any transforming of the received value before validating it. The value should match exactly what is in CHECKSUM_ALGORITHMS.